### PR TITLE
COV-369 PXP-6043 Fix/ui component

### PR DESCRIPTION
**Pull request by mfshao**
This PR fulfills https://occ-data.atlassian.net/browse/COV-369 and https://ctds-planx.atlassian.net/browse/PXP-6043

Fixed a CSS bug for data dict styles
Improved the logic of `RangeFilter` rerendering

Related to: https://github.com/uc-cdis/guppy/pull/92 and https://github.com/uc-cdis/data-portal/pull/701

### Bug Fixes
- Fixed a CSS bug causing data dict button to display incorrectly

### Improvements
- Change the `key` of `RangeFilter` so it would get rerendered more timely

### CODE REVIEW

File: stories/filters.js [hunk 1]
```diff
@@ -108,15 +107,15 @@ const fileSectionsWithTooltips = [
 ];
 
 const tabs = [
-  <FilterList key={0} sections={projectSections} />,
-  <FilterList key={1} sections={subjectSections} />,
-  <FilterList key={2} sections={fileSections} />,
+  <FilterList key={0} sections={projectSections} tierAccessLimit={1000} />,
+  <FilterList key={1} sections={subjectSections} tierAccessLimit={1000} />,
+  <FilterList key={2} sections={fileSections} tierAccessLimit={1000} />,
```
Comment (ZakirG) [inline stories/filters.js:112]: What's the reason for this change? Is the default tierAccessLimit value messing up the story view? Just curious

**Review by ZakirG — APPROVED** [review]

File: stories/filters.js (hunk 1 above)
Comment (mfshao) [reply stories/filters.js:112]: The `tierAccessLimit` should come from guppy but since there is no guppy so it needs to be put in manually. Otherwise the disabled filter in storybook will only have a "⬇️" on the side

Nothing major, just some cosmetic issues

**Review by em-ingram — APPROVED** [review]

File: src/components/filters/FilterSection/index.jsx [hunk 2]
```diff
@@ -339,7 +339,7 @@ class FilterSection extends React.Component {
                   // to rerender if the `reset` button is clicked.
                   // Each reset button click increments the counter and changes the key.
                   // See https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#recommendation-fully-uncontrolled-component-with-a-key
-                  const key = `${option.text}-${this.state.resetClickCounter}`;
+                  const key = `${option.text}-${option.min}-${option.max}-${lowerBound}-${upperBound}-${this.state.resetClickCounter}`;
```
[the remark this answers is not in the document]
Comment (mfshao) [reply src/components/filters/FilterSection/index.jsx:342]: true, this is to resolve problem 2

otherwise the filter can only rerender when the `resetClickCounter` got updated
